Workflow failure: Quarkus IT - FipsDistTest#testUnsupportedHttpsPkcs12KeyStoreInStrictMode

### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ci

### Describe the bug

[Quarkus IT (ubuntu-latest, zip)](https://github.com/keycloak/keycloak/actions/runs/7581549792/job/20649723076#logs)

`2024-01-19 09:25:26,452 ERROR [io.quarkus.vertx.http.runtime.QuarkusErrorHandler] (executor-thread-1) HTTP Request to //realms/master/ failed, error id: ff61811e-0fa3-4159-a6af-85130a71b2a8-93: java.lang.NullPointerException: Cannot invoke "io.quarkus.arc.ArcContainer.instance(java.lang.Class, java.lang.annotation.Annotation[])" because the return value of "io.quarkus.arc.Arc.container()" is null`

### Version

latest

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

-

### Anything else?

_No response_

@Pepo48 this is still related to #23786 - there is not an additional case to account for. The stop may not complete in time:

```
2024-01-19T09:25:38.0136572Z -- 	Suppressed: java.lang.RuntimeException: Failed to stop the server
2024-01-19T09:25:38.0137009Z -- 		at org.keycloak.it.utils.RawKeycloakDistribution.stop(RawKeycloakDistribution.java:170)
2024-01-19T09:25:38.0137579Z -- 		at org.keycloak.it.junit5.extension.KeycloakDistributionDecorator.stop(KeycloakDistributionDecorator.java:51)
2024-01-19T09:25:38.0138020Z -- 		at org.keycloak.it.junit5.extension.CLITestExtension.afterEach(CLITestExtension.java:182)
2024-01-19T09:25:38.0138715Z -- 		at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeAfterEachCallbacks$12(TestMethodTestDescriptor.java:260)
2024-01-19T09:25:38.0139699Z -- 		at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeAllAfterMethodsOrCallbacks$13(TestMethodTestDescriptor.java:276)
2024-01-19T09:25:38.0140265Z -- 		at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
2024-01-19T09:25:38.0140998Z -- 		at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeAllAfterMethodsOrCallbacks$14(TestMethodTestDescriptor.java:276)
2024-01-19T09:25:38.0141274Z -- 		at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
2024-01-19T09:25:38.0142003Z -- 		at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeAllAfterMethodsOrCallbacks(TestMethodTestDescriptor.java:275)
2024-01-19T09:25:38.0142662Z -- 		at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeAfterEachCallbacks(TestMethodTestDescriptor.java:259)
2024-01-19T09:25:38.0143210Z -- 		at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:144)
2024-01-19T09:25:38.0143331Z -- 		... 49 more
2024-01-19T09:25:38.0143671Z -- 	Caused by: java.lang.IllegalThreadStateException: process hasn't exited
2024-01-19T09:25:38.0143987Z -- 		at java.base/java.lang.ProcessImpl.exitValue(ProcessImpl.java:460)
2024-01-19T09:25:38.0144416Z -- 		at org.keycloak.it.utils.RawKeycloakDistribution.stop(RawKeycloakDistribution.java:166)
2024-01-19T09:25:38.0144535Z -- 		... 59 more
```

The test is waiting around 2+ minutes on a hung exit, then waiting an additional 10 seconds on the stop - but the default transaction timeout that we're waiting on is 3 minutes I believe. We'll need to increase the timeout, or surpress this failure as well.

--- a/quarkus/tests/junit5/src/main/java/org/keycloak/it/utils/RawKeycloakDistribution.java
+++ b/quarkus/tests/junit5/src/main/java/org/keycloak/it/utils/RawKeycloakDistribution.java
@@ -162,7 +162,8 @@ public final class RawKeycloakDistribution implements KeycloakDistribution {
                 destroyDescendantsOnWindows(keycloak, false);
 
                 keycloak.destroy();
-                keycloak.waitFor(DEFAULT_SHUTDOWN_TIMEOUT_SECONDS, TimeUnit.SECONDS);
+                // TODO: reconsider the hardcoded timeout once https://issues.redhat.com/browse/JBTM-3830 is pulled into Keycloak
+                keycloak.waitFor(180, TimeUnit.SECONDS);
                 exitCode = keycloak.exitValue();
             } catch (Exception cause) {
                 destroyDescendantsOnWindows(keycloak, true);

